### ISSUE TITLE
Enable-UcmO365Service returns random errors after enabling MCOStandard service

### ISSUE THREAD

**Issue by Atreidae** [Atreidae/UcmPSTools#11]
![image](https://user-images.githubusercontent.com/8736291/124696298-1e999680-df28-11eb-8176-541f301bf4a0.png)

Need to investigate this further, Enable-UCMO365Service was successfully called and returns OK (1) but then returns 2 unknown errors to the pipeline (2, 3)

Double checked the user both in OAC and PowerShell and the app is correctly assigned.
Admin Portal
![image](https://user-images.githubusercontent.com/8736291/124696481-8223c400-df28-11eb-8cb0-09bfae893736.png)

Native powershell
![image](https://user-images.githubusercontent.com/8736291/124696726-ef375980-df28-11eb-8fb5-6e809092a26f.png)

**Comment by Atreidae** [comment]
Upon running Test-UCMO365ServicePlan manually, it throws a ton of write warning errors as its referencing the old cmdlet name
![image](https://user-images.githubusercontent.com/8736291/124696928-4c330f80-df29-11eb-8a91-def6306d56e5.png)

However, this isn't called after setting the licence, so is not related.
The error message text matches the $AppEnabled flag test on line 165, I suspect we are stuck in a return loop.

**Comment by Atreidae** [comment]
Fixed Test-UCMO365ServicePlan Write-Log Cmdlet references, but still has issues returning result. opening new issue #12

**Comment by Atreidae** [comment]
This issue was caused by a few things.

1.  The cmdlet didn't actually check if we changed any services before attempting to write the changes back to O365 service plan so when a user had multiple licences (which is common) the cmdlet would throw an error for every serviceplan (licence) that didn't contain the required service

2. The Service was already enabled, thus there was nothing to "set"
3. The service isn't available in the assigned serviceplans

I've added a better error message indicating this as well as checks to see if we actually changed things before trying to write to Office365. 
This should also in theory improve the performance of this cmdlet considerably.
